apps/hash-graph/bin/cli/src/args.rs

```diff
@@ -6,6 +6,9 @@ use crate::subcommand::Subcommand;
 #[derive(Debug, Parser)]
 #[clap(version, author, about, long_about = None)]
 pub struct Args {
+    #[arg(long, env = "HASH_GRAPH_SENTRY_DSN")]
+    pub sentry_dsn: Option<Option<sentry::types::Dsn>>,
```

Comment:
Why this is a nested `Option`?

Reply:
I thought (from the clap documentation) that `Option<Option<T>>` would allow for empty values. I was wrong. Removed in [8e62433](https://github.com/hashintel/hash/pull/3091/commits/8e624331872d7bb6331f7e7bfd70a877aa182e66)

Reply:
For what an empty value would be used?

Reply:
The idea was to set in the .env (and in the production `docker-compose.prod.yml`) the `HASH_GRAPH_SENTRY_DSN` to an empty value (so `HASH_GRAPH_SENTRY_DSN=`), which would then render to `None` because it is an empty value, so sentry isn't activated (I mean an empty string will stop startup anyway as it isn't a valid DSN)